test/types/string/psahabu/perf/search.chpl

```diff
@@ -0,0 +1,64 @@
+use Assert;
+use Time;
+
+config const timing = true;
+config const n = 10000000;
+config const sourcePath = "moby.txt";
+
+var mobyFile = open(sourcePath, iomode.r);
+var mobyReader = mobyFile.reader();
+
+var Passage: [1..n] string;
+var word: string;
+for i in 1..n {
+  var read = mobyReader.read(word);
+  if !read {
+    mobyReader = mobyFile.reader();
+    continue;
+  }
+  Passage[i] = word;
+}
```

Comment:
Is this loop intended to read 'n' words, and start over if the file isn't long enough?